XR_APILAYER_NOVENDOR_toolkit/layer.cpp

```diff
@@ -394,8 +404,9 @@ namespace {
 
                             // This also means we need a non-sRGB type.
                             if (m_graphicsDevice->isTextureFormatSRGB(intermediateCreateInfo.format)) {
+                                    // good balance between visuals and perf
```

Comment:
Indentation looks a bit weird.

Reply:
I will review formatting on all files in the next commit.